src/main/java/internet/market/dao/OrderDao.java

```diff
@@ -0,0 +1,18 @@
+package internet.market.dao;
+
+import internet.market.model.Order;
+import internet.market.model.ShoppingCart;
+import java.util.List;
+import java.util.Optional;
+
+public interface OrderDao {
+    Order createOrder(ShoppingCart shoppingCart);
```

Comment:
```suggestion
    Order createOrder(Order order);
```
on dao layer we should just add ready order into our storage, additional logic is performed on service layer

Reply:
done